### PR TITLE
Updated old pages to acknowledge the existence of RPWS and Rebble's current status, also updated howto to point Android users to APKMirror for the app download

### PR DESCRIPTION
**Pull request by MisterSpak**
This PR is #51 but without the changes that require `auth` to be updated first. The following is now applicable to *this* PR.

Also includes @ja-besse's changes since last merge.

APKMirror isn't the most sightly site to direct users to, but it's at least trusted and unlikely to go anywhere. In the future I think it would be cleaner to (if we can) include a direct download link hosted in some location Rebble controls, on: the main page, in howto, and also possibly in the account section when users log in. For now, this will point Android users in the right direction. 

I have also included a new page, `unrebble`, which could at the very least be a troubleshooting tool for users that have config issues to easily be able to un-Rebble their Pebble app without wiping *all* data.

### CODE REVIEW

**Comment by MisterSpak** [comment]
@ishotjr

File: projects.md [hunk 1]
```diff
@@ -13,9 +13,8 @@ permalink: /projects/
 ## Appstore replacement
 
 * [The Rebble Store](https://github.com/pebble-dev/rebble-store) *(formerly The Panic Store)*
```
Comment (ishotjr) [inline projects.md]: If you're replacing *Panic* then this source link should be updated with the new source repo too - though a footnote about the prior effort would be nice, similar to *Android*?

File: projects.md [hunk 2]
```diff
@@ -26,8 +25,8 @@ permalink: /projects/
 
 ## Mobile Apps
 
-* [Android mobile app](https://github.com/pebble-dev/android-store-app)
-* iOS mobile app *(source link coming soon)*
+* [Official Pebble Android app](https://www.apkmirror.com/apk/pebble-technology-corp/pebble/pebble-4-4-2-1405-62d45d7d7-endframe-release/pebble-4-4-2-1405-62d45d7d7-endframe-android-apk-download/) *(For use with Rebble Web Services. Site unaffiliated with Rebble, go to rebble.io/howto for full installation and setup instructions)*
```
Comment (ishotjr) [inline projects.md]: why not make this a link? :bowtie: 💭 🔗

File: status.md [hunk 3]
```diff
@@ -45,17 +45,17 @@ developments occur.
...
-| timeline-sync.<span class="grayed">getpebble.com</span><br/>timeline-api.<span class="grayed">getpebble.com</span> | Timeline servers. | Will continue to be maintained for an indefinite amount of time. Community replacement possible, but very low-priority. |
+| lp.<span class="grayed">getpebble.com</span> | Language packs. | Functionality supplanted by Rebble Web Services. |
+| apps.<span class="grayed">getpebble.com</span><br/>appstore-api.<span class="grayed">getpebble.com</span> | App Store. | Functionality supplanted by Rebble Web Services (apps.rebble.io). |
+| help.<span class="grayed">getpebble.com</span><br/>appstore-api.<span class="grayed">getpebble.com</span> | Support pages. | Mirror available [here](http://pebble-help-legacy.rebble.io/help.getpebble.com/indexd41d.html). |
+| cloudpebble.net | Cloud-based IDE. | CloudPebble is open source and will continue as a community project once it once it goes down. **_However_**, you should definitely [export your CloudPebble projects for local safekeeping!](https://www.reddit.com/r/pebble/comments/5h61om/bulk_cloudpebble_project_export/) Unclear if it'll ever come back as a part of Rebble Web Services. |
```
Comment (ishotjr) [inline status.md]: I'm not crazy about this language - can we come up with something less casual/more "scientific" ? ⚗️

File: faq.md [hunk 4]
```diff
@@ -8,18 +8,22 @@ permalink: /faq/
...
+Well, right here! At least, for frequently asked questions about *The Rebble Alliance*. For frequently asked questions about how to use the Rebble Web Services with your Pebble smartwatch, head to [the setup and troubleshooting how-to](/howto) and scroll down.
+
 ### What is Rebble?
 
 Rebble began as a supplement to the [official Pebble documentation](https://developer.pebble.com/).  After [Pebble ceased operations](https://blog.getpebble.com/2016/12/07/fitbit/), it morphed into the unofficial spearhead organization for continuing the advancement of the Pebble platform via the amazing Pebble developer community.  
 
 ### What does Rebble do?
 
-Our present focus is on replacing key pieces of the Pebble ecosystem: the appstore, firmware, and mobile apps.  At a higher level, our goal is to maintain and advance Pebble functionality, in the absence of *Pebble Technology Corp*.  
+Our present focus is on replacing key pieces of the Pebble ecosystem: the appstore and related web services, and later on down the road, potentially even firmware and mobile apps.  At a higher level, our goal is to maintain and advance Pebble functionality, in the absence of *Pebble Technology Corp*.
```
Comment (ishotjr) [inline faq.md]: Not super clear on the motivation behind this tweak? 🔎

File: status.md (hunk 3 above)
Comment (MisterSpak) [reply status.md]: To be completely honest I have no idea about the specifics - I didn't want to go out on a limb and say anything that wasn't correct - so some pointers here from someone who knows more about the status and integration of all those moving parts would be very helpful!

File: faq.md (hunk 4 above)
Comment (MisterSpak) [reply faq.md]: The language change is meant to make the app store and other web services distinct from RebbleOS and any potential mobile apps projects. One is already being used by thousands of people, the others are much further ahead. Also, a lot of people seem to come to the Discord server or post in the subreddit thinking Rebble is all about firmware, but that hasn't been the case in a long time and might not be the case for a long time either.

File: projects.md (hunk 2 above)
Comment (MisterSpak) [reply projects.md]: Great idea! I should have done that to begin with. I'll make that change when I have the chance.

File: projects.md (hunk 1 above)
Comment (MisterSpak) [reply projects.md]: Ah, I didn't know there was a new repo for that! I'll fix that ASAP.

Comment (MisterSpak) [reply projects.md]: I just realized I can't seem to find the new repo. Is there a link I can find that at, or the name of the repo?

File: projects.md (hunk 2 above)
Comment (MisterSpak) [reply projects.md]: Done!

File: faq.md (hunk 4 above)
Comment (MisterSpak) [reply faq.md]: I can change it back if anyone disagrees.

**Pull request merged**